src/app/pages/display/display.component.html

```diff
@@ -198,3 +198,20 @@
     <mat-spinner></mat-spinner>
   </div>
 </ng-template>
+
+<ng-template #countdown >
+<section class="countdown p-4 shadow-2">
+  <div>
+    <mat-progress-spinner
+        class="progressBar"
+        [mode]="mode"
```

Comment:
```suggestion
        mode="determinate"
```

You can remove the brackets to use a literal value directly.